src/node.cc

```diff
@@ -3224,6 +3224,21 @@ void SetupProcessObject(Environment* env,
                     Integer::New(env->isolate(),
                                  v8::kPromiseHandlerAddedAfterReject));
 
+  // process._debugOptions
+  Local<Object> debugOptions = Object::New(env->isolate());
+  READONLY_DONT_ENUM_PROPERTY(process,
+                              "_debugOptions",
```

Comment:
A few comment (don't fix then until we get more feedback)
1. There is preference to have this in `src/node_constants.cc` but I'm not sure how easy that is to move it
2. I would call it `_cliDebugOptions` and add `enabled`.
3. Maybe there's a more efficient way to set these since they are constants?